rdflib/term.py

```diff
@@ -370,6 +395,15 @@ def de_skolemize(self) -> "BNode":
             raise Exception("<%s> is not a skolem URI" % self)
 
 
+# Cannot import Namespace/XSD because of circular dependencies
+_XSD_PFX = "http://www.w3.org/2001/XMLSchema#"
+_RDF_PFX = "http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+
+_RDF_LANGSTRING = URIRef(_RDF_PFX + "langString")
```

Comment:
It might be more useful to make `_RDF_LANGSTRING` a class attribute for Literal. Then someone can change the default datatype for langstring by using a childclass instead of changing it programwide.
But im not sure